sql/core/v1.2/src/main/java/org/apache/spark/sql/execution/datasources/orc/OrcColumnVector.java

```diff
@@ -42,6 +43,7 @@
   private DecimalColumnVector decimalData;
   private TimestampColumnVector timestampData;
```

Comment:
does ORC have `DateColumnVector`?

Reply:
Yes, it does, but:
1. `DateColumnVector` doesn't have a method similar to `asScratchTimestamp` in `TimestampColumnVector`
2. We don't need to build `java.sql.Date` from serialized days to perform rebasing. It is unnecessary overhead.